framework/src/modules/interoperability/schema.ts

```diff
@@ -468,3 +468,112 @@ export const sidechainTerminatedCCMParamsSchema = {
 		},
 	},
 };
+
+export const nameSchema = {
+	$id: 'modules/interoperability/name',
+	type: 'object',
+	required: ['name'],
+	properties: {
+		name: {
+			dataType: 'string',
+			fieldNumber: 1,
+		},
+	},
+};
+
+export const chainIDSchema = {
+	$id: 'modules/interoperability/name',
+	type: 'object',
+	required: ['ID'],
+	properties: {
+		ID: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+	},
+};
+
+export const validatorsSchema = {
+	$id: 'modules/interoperability/validators',
+	type: 'object',
+	required: ['activeValidators', 'certificateThreshold'],
+	properties: {
+		activeValidators: {
+			type: 'array',
+			fieldNumber: 1,
+			items: {
+				type: 'object',
+				required: ['blsKey', 'bftWeight'],
```

Comment:
Not adding this check here since `activeValidators` can be both mainchain & sidechain. But added it in the schema `mainchainRegParams` for `mainchainValidators`. Please have a look at the latest commits ([1](https://github.com/LiskHQ/lisk-sdk/pull/7147/commits/bbf16afa7a0a87a7394c67020c20a58fe90644b7) & [2](https://github.com/LiskHQ/lisk-sdk/pull/7147/commits/6abbe9b71049cdcd1774f202ac2476ea555f36bc)) for this change.